components/01-globals/grid/GridCollumn.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div :class="['grid__column', width ? `grid__column--${width}` : '']">
+  <div :class="['grid__column', width && `grid__column--${width}`]">
```

Comment:
This is weird and probably doesn't work anymore, b/c `&&` return Boolean, while we need String.

Reply:
It works fine this way :)